### ISSUE TITLE
Pdb file is included when downloading version 7.3.1 from asset store

### ISSUE THREAD

**Issue by adarapata** [ssannandeji/Zenject-2019#575]
I am sorry if it is difficult to understand because I am using Google translation.

There are three files under `Plugins/Zenject/Source/Usage/`.
- link.xml
- Zenject-usage.dll.mdb
- Zenject-usage.pdb

I thought they would not be necessary, but I would like you to tell me if there is a purpose.

**Comment by svermeulen** [comment]
Yeah agreed there's no reason we should include the pdb and mdb.  But the link.xml is needed to ensure there's no AOT issues

**Comment by adarapata** [comment]
I understand, thank you. :)
